src/main/java/org/galatea/starter/entrypoint/SettlementRestController.java

```diff
@@ -34,34 +37,60 @@
 @RestController
 public class SettlementRestController {
 
-  @NonNull
-  SettlementService settlementService;
-
-  public static final String SETTLE_MISSION_PATH = "/settlementEngine";
-  public static final String GET_MISSION_PATH = SETTLE_MISSION_PATH + "/mission/";
-
-  @PostMapping(value = SETTLE_MISSION_PATH, consumes = {MediaType.APPLICATION_JSON_VALUE})
-  public ResponseEntity<Set<String>> settleAgreement(
-      @RequestBody final List<TradeAgreement> agreements) {
-    Set<Long> missionIds = settlementService.spawnMissions(agreements);
-    Set<String> missionIdUris =
-        missionIds.stream().map(id -> GET_MISSION_PATH + id).collect(Collectors.toSet());
-
-    return ResponseEntity.accepted().body(missionIdUris);
-  }
-
-  @GetMapping(value = GET_MISSION_PATH + "{id}", produces = {MediaType.APPLICATION_JSON_VALUE})
-  public ResponseEntity<SettlementMission> getMission(@PathVariable final Long id) {
-
-    Optional<SettlementMission> msn = settlementService.findMission(id);
-
-    if (msn.isPresent()) {
-      return ResponseEntity.ok(msn.get());
-    }
-
-    return new ResponseEntity<>(HttpStatus.NOT_FOUND);
-  }
-
-
+	@NonNull
+	@Autowired
```

Comment:
Is there a reason to add this in?  master no longer has this since it's not required.